spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -64,19 +64,22 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
 
   object InsertIntoHadoopFsRelationHarvester extends Harvester[InsertIntoHadoopFsRelationCommand] {
     override def harvest(node: InsertIntoHadoopFsRelationCommand, qd: QueryDetail): Seq[AtlasEntity] = {
-      // source tables entities
+      // source tables/files entities
       val tChildren = node.query.collectLeaves()
       val inputsEntities = tChildren.map {
         case r: HiveTableRelation => tableToEntities(r.tableMeta)
         case v: View => tableToEntities(v.desc)
-        case l: LogicalRelation => tableToEntities(l.catalogTable.get)
+        case l: LogicalRelation =>
```

Comment:
Please split into two `case` statements.

Reply:
Ping, @weiqingy . This is the last ongoing PR.

Reply:
I'll update this pr very soon

Reply:
Done

Reply:
Ur, what I meant by `splitting case statement` is the following.
```scala
case l: LogicalRelation if l.catalogTable.isDefined =>
  l.catalogTable.map(tableToEntities(_)).get
case l: LogicalRelation =>
  l.relation match {
    case r: FileRelation => r.inputFiles.map(external.pathToEntity).toSeq
    case _ => Seq.empty
  }
```